src/nnvm/legacy_op_util.cc

```diff
@@ -328,10 +424,14 @@ void RegisterLegacyOpProp() {
     op.set_attr<nnvm::FMutateInputs>("FMutateInputs", OpPropMutateInputs);
     op.set_attr<nnvm::FInplaceOption>("FInplaceOption", OpPropInplaceOption);
     op.set_attr<FResourceRequest>("FResourceRequest", OpPropResourceRequest);
-    op.set_attr<FCreateLayerOp>("FCreateLayerOp", OpPropCreateLayerOp);
+    op.set_attr<FExecType>("FExecType", OpExecType);
+    op.set_attr<FCreateOpState>("FCreateOpState", OpPropCreateLayerOp);
+    op.set_attr<FStatefulCompute>("FStatefulCompute<cpu>", LegacyOpForward);
+    op.set_attr<FStatefulCompute>("FStatefulCompute<gpu>", LegacyOpForward);
     if (reg->key_var_num_args.length() != 0) {
```

Comment:
How to set attr for `FStatefulComputeEx<xpu>` for legacy ops?  Do we have to modify the interface of operator class to have Forward/Backward with NDArray inputs & outputs?